src/lambdas/initiateAthenaQuery/handler.ts

```diff
@@ -72,14 +75,6 @@ const confirmQuerySqlGeneration = async (
     await updateZendeskTicketById(zendeskId, querySql.error, 'closed')
     throw new Error(querySql.error)
   }
-
-  if (querySql.sql) {
```

Comment:
Would it not be useful to know what SQL was generated? This would help debugging queries, and nothing sensitive in the query itself!

Reply:
We took this out on the basis that the query SQL can be seen in the Athena console, even if the query fails, and so it would be possible to find it without needing to log it out. It's a matter of convenience perhaps - do we want to log information that can be found elsewhere?

Reply:
I think the consensus following our meeting earlier is that we can look this up in Athena, as Tom said, so I'm resolving this.